src/meshcat/geometry.py

```diff
@@ -296,6 +301,19 @@ def pack_numpy_array(x):
     }
 
 
+def data_from_stream(stream):
+    if sys.version_info >= (3, 0):
+        if type(stream) == BytesIO:
```

Comment:
Should we also support subtypes of `BytesIO` and/or `StringIO` by using `isinstance` instead of type equality?

Reply:
OK. That's now changed in the latest version.